### PR TITLE
Bug fixes in calc_signal and get_sample

### PR DESCRIPTION
**Pull request by lasofivec**
Found a couple of bugs:

- when sampling a line with variable resolution (`los_get_sample_core_var_res`): `+` instead of `*` in formula.
- when integrating matrix in `LOS_calc_signal`: forgot to transpose matrix. 
- when using "minimize=hybrid" and "minimize=memory": wrong limits of LOS.

Additionally, I also took out the file `_fast_sum.c` in order to have less "low level" utilities and have a more maintainable code even if we lose a small percentage of optimization...

### CODE REVIEW

**Comment by lasofivec** [comment]
I hesitated about doing this but: 
- There are already a lot of test cases doing this.
- LOS_get_sample tests cases are the slowest
- Adding the plot_custom_emissivity.py to the tutorials will provide a sort of unit-test

**Comment by Didou09** [comment]
You're right, 
I didn't add an extra test case, I just added an extra line (np.allclose()) to already existing cases, only for 1D cameras (100 LOS)

**Comment by Didou09** [comment]
I am also creating a new tests repository specifically to run tutorials (agree ?)

**Comment by Didou09** [comment]
Fies, in devel, Issue #247

File: examples/tutorials/tuto_plot_create_geometry.py [hunk 1]
```diff
@@ -9,6 +9,12 @@
 import matplotlib.pyplot as plt
 import tofu.geom as tfg
 
+import os
+import tofu as tf
+print(os.getcwd())
+print(tf.__version__)
+print(tf.__path__)
+
```
[the remark this answers is not in the document]
Comment (Didou09) [reply examples/tutorials/tuto_plot_create_geometry.py]: These lines were added for debugging, I forgot to remove them, yes you can delete them

**Pull request merged**